packages/eslint-plugin/tests/rules/lines-around-comment.test.ts

```diff
@@ -791,31 +761,6 @@ enum A {
       ],
       errors: [{ messageId: 'after', type: AST_TOKEN_TYPES.Line, line: 3 }],
     },
-    {
-      code: unIndent`
-enum A {
-  a,
-  /* block
-     comment */
-}
-`,
-      output: unIndent`
-enum A {
-  a,
-  /* block
-     comment */
-
-}
-`,
-      options: [
-        {
-          beforeBlockComment: false,
-          afterBlockComment: true,
-          allowEnumEnd: false,
-        },
-      ],
-      errors: [{ messageId: 'after', type: AST_TOKEN_TYPES.Block, line: 3 }],
-    },
```

Comment:
This rule is being removed in #8833 anyway. I didn't feel like fixing up bugs in it. 🔪